src/NuGet.Core/NuGet.Commands/Utility/SdkAnalysisLevelMinimums.cs

```diff
@@ -15,6 +15,11 @@ internal static class SdkAnalysisLevelMinimums
         /// </summary>
         internal static readonly NuGetVersion HttpErrorSdkAnalysisLevelMinimumValue = new("9.0.100");
 
+        /// <summary>
+        /// Minimum SDK Analysis Level required for warning for packages and projects that cannot be pruned.
+        /// </summary>
+        internal static readonly NuGetVersion PruningWarnings = new("10.0.100");
```

Comment:
Given my experience with NuGetAuditMode, I'm now a huge opponent of SdkAnalysisLevel.  It doesn't work the way a lot of customers expect it to, and it doesn't work in the way that our division's leadership want us to provide backwards compatibility to customers. I think SdkAnalysisLevel doesn't work and should be considered a failed experiment we no longer use.

Unfortunately, I also don't have a suggestion on how we'll ever turn on new features by default. It's something we're going to have to have discussions about (and when we have a proposal, take it to leadership) early next year.

Reply:
I don't know if we should give up on SDKAnalysisLevel after a single release, but we should certainly be looking for something that allows to enable defaults for *every* scenario.

Your feedback is similar to Jeff's and I have created a follow-up that I'll make part of the backlog of PrunePackageReference and something we want to figure out before the official roll-out enabled by default in whatever scenario we deem we want it: https://github.com/NuGet/Client.Engineering/issues/3111